test/draw/RasterCanvasTest.ooc

```diff
@@ -3,6 +3,7 @@ use ooc-math
 use ooc-draw
 use ooc-draw-gpu
 use ooc-opengl
+import RasterCanvas
```

Comment:
I had a compilation error. I guess it does not matter, I have another commit pending where this line is removed.